variety/__init__.py

```diff
@@ -152,47 +132,39 @@ def check_quit():
         GObject.idle_add(VARIETY_WINDOW.on_quit)
     Util.start_force_exit_thread(10)
 
-
-def monkeypatch_ssl():
```

Comment:
It should also be OK now to remove the ugly verify=False option in Util.request that was there specifically to remove issues with SSL with Wallhaven

Reply:
(but needs testing)